<launch>
  <!-- Load all of the motor controllers -->
  <rosparam file="$(find tfr_control)/config/controllers.yaml" command="load"/>

  <!-- Load the controller manager plugin for the drivebase -->
  <node name="controller_launcher" pkg="tfr_control" type="controller_launcher" />

  <!-- Spawn the controllers -->
  <node name="controller_spawner" pkg="controller_manager" type="spawner"
          args="joint_state_controller
                left_tread_velocity_controller
                right_tread_velocity_controller
                bin_position_controller
                arm_controller
                arm_end_controller --shutdown-timeout 1"/>
</launch>